docs/components/customization/user-profile.mdx

```diff
@@ -116,301 +34,321 @@ All props below are required.
+  );
+
+  export default UserProfilePage;
+  ```
+
+  ```tsx filename="/app/components/Header.tsx"
+  "use client";
+
+  import { UserButton } from "@clerk/nextjs";
+
+  const DotIcon = () => {
+    return (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 512 512"
+        fill="currentColor"
+      >
+        <path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512z" />
+      </svg>
+    )
+  }
+
+  const Header = () => (
+    <header>
+      <UserButton afterSignOutUrl='/'>
+        <UserButton.UserProfileLink
+          label="Homepage"
+          url="/"
+          labelIcon={<DotIcon />}
+        />
+      </UserButton>
+    </header>
+  );
+
+  export default Header;
+  ```
+</CodeBlockTabs>
 
-<Tabs type="framework" items={["Next.js", "React", "Remix"]}>
-  <Tab>
-    `<UserProfile.Page />` and `<UserProfile.Link />` can be rendered **only on the client**. For Next.js applications using App Router, the `"use client";` directive must be added.
-
-    <CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-      ```jsx filename="/app/user-profile/[[...user-profile]]/page.tsx"
-      "use client";
-
-      import { UserProfile } from "@clerk/nextjs";
-      import { MyCustomPageContent } from "../components";
-      import { CustomIcon, Icon } from "../icons";
-
-      const UserProfilePage = () => (
-        <UserProfile path="/user-profile" routing="path">
-          <UserProfile.Page label="Custom Page" url="custom" labelIcon={<CustomIcon />}>
-              <MyCustomPageContent />
-          </UserProfile.Page>
-          <UserProfile.Link label="Homepage" url="/home" labelIcon={<Icon />} />
-          <UserProfile.Page label="account" />
-          <UserProfile.Page label="security" />
-        </UserProfile>
-      );
+## Advanced use cases
 
-      export default UserProfilePage;
-      ```
-
-      ```jsx filename="/pages/user-profile/[[...index]].tsx"
-      import { UserProfile } from "@clerk/nextjs";
-      import { MyCustomPageContent } from "../components";
-      import { CustomIcon, Icon } from "../icons";
-
-      const UserProfilePage = () => (
-        <UserProfile path="/user-profile" routing="path">
-          <UserProfile.Page label="Custom Page" url="custom" labelIcon={<CustomIcon />}>
-              <MyCustomPageContent />
-          </UserProfile.Page>
-          <UserProfile.Link label="Homepage" url="/home" labelIcon={<Icon />} />
-          <UserProfile.Page label="account" />
-          <UserProfile.Page label="security" />
-        </UserProfile>
-      );
+### Reorder default routes
 
-      export default UserProfilePage;
-      ```
-    </CodeBlockTabs>
-  </Tab>
+If you want to reorder the default routes (`Account` and `Security`) in the `<UserProfile />` navigation sidebar, you can use the `<UserProfile.Page />` component with the `label` prop set to `'account'` or `'security'`. This will target the existing default page and allow you to rearrange it.
```

Comment:
```suggestion
If you want to reorder the default routes (`Account` and `Security`) in the `<UserProfile />` navigation sidebar, add two `<UserProfile.Page />` components. each with the `label` prop set to either `'account'` or `'security'`. The order in which you render them will be the order in which they appear.
```

My suggestion may not be right, but I didn't quite understand this when I first read it. I _think_ what's happening here is you can label each `<UserProfile.Page>` and adjust the order by swapping their position. If that's so, it might help to say that more explicitly. Would love another opinion though as well cc @jescalan @royanger

Reply:
>`<UserProfile.Page />` components.
You added a period after the components, preceded by "each with"

Reply:
just updated the copy - lmk what you think!